src/LogViewerService.php

```diff
@@ -36,9 +36,9 @@ protected function getFilePaths(): array
     }
 
     /**
-     * @return Collection|LogFile[]
```

Comment:
I'd like to leave the `LogFile[]` annotation here because it helps PHPStorm and other IDEs understand that this `Collection` has instances of `LogFile` as values.

Without it, PHPStorm thinks this is a collection of `mixed` values. With this, it knows it's a collection of `LogFile` values.